### ISSUE TITLE
Adjust realtime weapon switcher selection to begin from existing setting.

### ISSUE THREAD

**Issue by pdatumoj** [pdatumoj/slashscript#4]
The realtime weapon switcher selectors should, in the case of the toggle modes, start from the existing setting for the loadout ... maybe?  This would depend on #1 and #3 .

**Comment by pdatumoj** [comment]
Created a test implementation of this and found the UX impact was more harmful than helpful - this is now wontfix.
